hathor/indexes/rocksdb_tx_group_index.py

```diff
@@ -0,0 +1,141 @@
+# Copyright 2021 Hathor Labs
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from abc import abstractmethod
+from typing import TYPE_CHECKING, Iterable, Optional, Sized, Tuple, TypeVar
+
+from structlog import get_logger
+
+from hathor.indexes.rocksdb_utils import RocksDBIndexUtils
+from hathor.indexes.tx_group_index import TxGroupIndex
+from hathor.transaction import BaseTransaction
+
+if TYPE_CHECKING:  # pragma: no cover
+    import rocksdb
+
+logger = get_logger()
+
+KT = TypeVar('KT', bound=Sized)
+
+
+class RocksDBTxGroupIndex(TxGroupIndex[KT], RocksDBIndexUtils):
+    """ Index of inputs/outputs by address.
```

Comment:
Outdated docstring with address mention. The same for some methods

Reply:
Fixed! Thanks!